libs/glib2/patches/003-valgrind.h-mips16-fix.patch

```diff
@@ -0,0 +1,11 @@
+--- a/glib/valgrind.h	2019-12-12 14:53:26.000200499 +0100
++++ b/glib/valgrind.h	2019-12-12 14:49:45.056163300 +0100
+@@ -157,7 +157,7 @@
+ #  define PLAT_s390x_linux 1
+ #elif defined(__linux__) && defined(__mips__) && (__mips==64)
+ #  define PLAT_mips64_linux 1
+-#elif defined(__linux__) && defined(__mips__) && (__mips!=64)
++#elif defined(__linux__) && defined(__mips__) && (__mips!=64) && !defined(__mips16)
```

Comment:
Can be simplified to && __mips==32 . No need for the __mips16 . mips16 turns __mips into 1 oddly enough.

Reply:
I resubmitted upstream: https://sourceforge.net/p/valgrind/mailman/message/36880442/

Reply:
This ended up upstream: https://sourceware.org/git/?p=valgrind.git;a=commit;h=2bddca658976a69e9e24438891295e2af1dd3c09

Reply:
Ignore my comments. This is fine.